WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -217,6 +217,11 @@ public void Initialize(NodesCollection nodes, WasabiSynchronizer synchronizer, U
 
 					UpdateAvailable = !x.ClientUpToDate;
 					CriticalUpdateAvailable = !x.BackendCompatible;
+
+					if (!x.LegalDocsUpToDate)
+					{
+						// TODO: ensure that the user agreed with the terms and conditions.
```

Comment:
Here we can terminate the user workflow with the agreement of the new LegalDocs. Questions to be answered.

- What if we get the update while CoinJoining? 
  - Shall I open a new screen and not letting the user do anything else until agreed.
  - Shall I make a notification on the status bar?